tests/std/tests/P0896R4_views_empty/test.cpp

```diff
@@ -50,6 +50,15 @@ constexpr bool test_one_type() {
     static_assert(noexcept(R::empty()));
     static_assert(noexcept(ranges::empty(r)));
 
+#if _HAS_CXX23
+    // validate cbegin, and cend

```

Comment:
Nitpicks:
* Unnecessary comma.
* The previous comments also say "members".
* These are inherited from `view_interface`, which is why they have to be called with `r.` instead of `R::`. It seems reasonable to note this in the comment, and update the final comment.